util/exponential-backoff.js

```diff
@@ -47,7 +46,7 @@ async function exponentialBackoff (url, requestOptions, retryOptions) {
  * @private
  */
 async function _exponentialBackoffHelper (url, requestOptions, retryOptions) {
-  const { maxRetries, retryMultiplier, retryCount } = retryOptions
+  const { maxRetries, retryMultiplier, retryCount = DEFAULT_RETRY_COUNT } = retryOptions
```

Comment:
minor comment: I feel that we can get rid of DEFAULT_RETRY_COUNT at this point and just set it to 0